exercises/C-comparison-operators/exercise.js

```diff
@@ -7,22 +7,24 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount + mentorCount; // finish this statement
```

Comment:
`moreStudentsThanMentors` should equal either true of false, with having `studentCount + mentorCount` here you'll end up with a number instead